infrastructure/helm-chart/templates/ingress.yaml

```diff
@@ -297,7 +288,7 @@ spec:
                 name: sources-chatplugin
                 port:
                   number: 80
-          - path: /
+          - path: /ui/chatplugin
```

Comment:
I am assuming that there would be a conflict with the `ui` pod here, so I would give this a different prefix